### PR TITLE
MAT-7995 filter out custom errors

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java [hunk 1]
```diff
@@ -111,4 +111,20 @@ private List<CqlCompilerException> filterBySyntax(
...
+   * This is done in cql-antlr-parse, so on the frontend we don't want a duplicate error message
+   * therefore we are filtering it out here.
+   */
+  private List<CqlCompilerException> filterOutCustomErrors(
+      List<CqlCompilerException> filteredCqlTranslatorExceptions) {
+    return filteredCqlTranslatorExceptions.stream()
+        .filter(
+            cqlCompilerException ->
+                !"no viable alternative at input 'define :'"
+                    .equalsIgnoreCase(cqlCompilerException.getMessage()))
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java]: How about checking for contains instead of equals? The presence of the colon leads me to believe the incoming string may have more text.

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/mat/cql_elm_translation/service/filters/CqlTranslatorExceptionFilter.java]: @jkotanchik-SB just made the change, thanks!

**Review by jkotanchik-SB — APPROVED** [review]
